pkg.go

```diff
@@ -115,6 +115,7 @@ type pkgMetadata struct {
 type pkgSpine struct {
 	Items []pkgItemref `xml:"itemref"`
 	Toc   string       `xml:"toc,attr"`
+	Ppd   string       `xml:"page-progression-direction,attr"`
```

Comment:
Could you use `omitempty` here (https://github.com/bmaupin/go-epub/blob/master/pkg.go#L81)? I think that should prevent this from being added if it's not set, which should also keep from breaking TestEpubWrite (https://travis-ci.org/bmaupin/go-epub/builds/183031639)